src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import cn from 'classnames';
+
+export const Person = ({ person: {
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+} }) => (
+  <section className={cn('Person')}>
```

Comment:
library classnames need to use for making dynamic classnames, in this case, you have a static classname
so, `cn` is redundant here